src/psl.c

```diff
@@ -1588,7 +1588,7 @@ static int isip(const char *hostname)
 	INT size  = sizeof(addr);
 	INT size6 = sizeof(addr6);
 
-	if (!MultiByteToWideChar(CP_UTF8, 0, hostname, -1, wName, sizeof(wName)/sizeof(WCHAR)))
+	if (!MultiByteToWideChar(CP_UTF8, 0, hostname, -1, wName, countof(wName)))
```

Comment:
please merge with the previous commit.

Reply:
Sorry, please excuse my ignorance... what do you mean with merge with previous commit?
Is it a thing that I can do with github web interface?

Reply:
I guess you can't use Github for that. You can do that by using 'git rebase -i' (you find examples on the web) and after that you force push, Github then amends your PR automatically.

Reply:
Just don't mind, I'll merge all your commits into one before adding your changes to master.